modules/ROOT/pages/platform/api/authentication.adoc

```diff
@@ -7,7 +7,7 @@ label:AuraDS-Enterprise[]
 
 [TIP]
 ====
-The Aura API is currently in Public Beta for Aura Enterprise customers only. If you wish to participate in the Public Beta, please contact your Customer Success Manager.
+The Aura API is currently in beta for Aura Enterprise customers only. If you wish to participate in the beta, please contact your Customer Success Manager.
```

Comment:
I wonder if we should add "participate in the beta _testing_"?

Reply:
@Paul-Blewett are you happy to mention the term test here?

Reply:
Or we could say _beta phase_ or _beta stage_ even.

Reply:
i like stage so will go with that